model_compression_toolkit/target_platform_capabilities/target_platform/targetplatform2framework/attach2fw.py

```diff
@@ -0,0 +1,47 @@
+from typing import Dict, Tuple, List, Any, Optional
+
+from model_compression_toolkit import DefaultDict
+from model_compression_toolkit.target_platform_capabilities.schema.mct_current_schema import TargetPlatformModel
+from model_compression_toolkit.target_platform_capabilities.target_platform import TargetPlatformCapabilities, \
+    OperationsSetToLayers
+
+
+class TpcAttach2Fw:
+
+    def __init__(self):
+        self._opset2layer = None
+
+        # we provide attributes mapping that maps each layer type in the operations set
+        # that has weights attributes with provided quantization config (in the tp model) to
+        # its framework-specific attribute name.
+        # note that a DefaultDict should be provided if not all the layer types in the
+        # operation set are provided separately in the mapping.
+        self._opset2attr_mapping = None
+
+    def attach(self, tpc_model: TargetPlatformModel,
+               custom_opset2layer: Dict[str, Tuple[List[Any], Optional[Dict[str, DefaultDict]]]] = None
+               ) -> TargetPlatformCapabilities:
+
```

Comment:
Need to add documentation

Reply:
"""
    Attach a TargetPlatformModel, which describes platform capabilities, to specific framework operators.
    
    Args:
        tpc_model: A TargetPlatformModel object.
        custom_opset2layer: An optional dictionary to add or override the built-in framework operator sets. 
            This dictionary maps a unique operator set name to a pair consisting of:
            1. A list of framework operators.
            2. An optional mapping of operator attribute names.

    Returns:
        A TargetPlatformCapabilities object.
    """